client/src/components/MeetingExperience.test.tsx

```diff
@@ -187,10 +187,13 @@ describe('MeetingExperience', () => {
 
     meetingExperience.update();
 
-    const survey = meetingExperience.find(Survey);
+    expect(setRenderPostCallMock).toBeCalled();
     const callWithChatComposites = meetingExperience.find(CallWithChatComposite);
-    expect(callWithChatComposites.length).toBe(0);
-    expect(survey.length).toBe(1);
-    expect(callWithChatComposites.length).toBe(0);
+    expect(callWithChatComposites.length).toBe(1);
+    const parentDiv = callWithChatComposites.parent();
+    expect(parentDiv.props().style.display).toBe('none');
+
+    const iframe = meetingExperience.find('iframe');
```

Comment:
Can we find the Survey component directly here?
```
const surveyComponent = meetingExperience.find(Survey);
expect(surveyComponent.length).toBe(1);
```

That way the iframe is abstracted from this test.

Reply:
i see we do the inverse of this in the previous test above line 160, so making this consistent makes sense

Reply:
made this change